ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -19,6 +19,7 @@ ${NOTEBOOK_IMAGE_STREAM_NAME}   %{NOTEBOOK_IMAGE_STREAM_NAME=s2i-generic-data-sc
 Run TestMNISTPyTorchMCAD E2E test
     [Documentation]    Run Go E2E test: TestMNISTPyTorchMCAD
     [Tags]  ODS-2543
+    ...     Sanity
     ...     Tier2
```

Comment:
Please remove "Tier2" tag as it will be run as part of "Sanity" tests. It doesn't make sense to run the same test twice.

Reply:
I would rather prefer to keep it, so if you check Tier2 results you can see all the relevant tests.